src/common/transformations/tests/op_conversions/scaled_dot_product_decomposition_test.cpp

```diff
@@ -0,0 +1,159 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include "common_test_utils/ov_test_utils.hpp"
+#include "openvino/core/model.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/broadcast.hpp"
+#include "openvino/op/concat.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/convert_like.hpp"
+#include "openvino/op/divide.hpp"
+#include "openvino/op/gather.hpp"
+#include "openvino/op/greater_eq.hpp"
+#include "openvino/op/logical_not.hpp"
+#include "openvino/op/matmul.hpp"
+#include "openvino/op/multiply.hpp"
+#include "openvino/op/range.hpp"
+#include "openvino/op/scaled_dot_product_attention.hpp"
+#include "openvino/op/select.hpp"
+#include "openvino/op/shape_of.hpp"
+#include "openvino/op/softmax.hpp"
+#include "openvino/op/sqrt.hpp"
+#include "openvino/op/squeeze.hpp"
+#include "openvino/op/transpose.hpp"
+#include "openvino/op/unsqueeze.hpp"
+#include "transformations/op_conversions/scaled_dot_product_attention_decomposition.hpp"
+
+using namespace ov;
+using namespace testing;
+
+const std::shared_ptr<ov::Node> scaled_dot_product_attention_decomposition(
+    const std::shared_ptr<ov::Node> query,
+    const std::shared_ptr<ov::Node> key,
+    const std::shared_ptr<ov::Node> value,
+    const std::shared_ptr<ov::Node> attention_mask,
+    const std::shared_ptr<ov::Node> scale,
+    const bool casual);
+
+TEST_F(TransformationTestsF, ScaledDotProductAttentionDecompositionStatic) {
+    const PartialShape query_key_value_mask_scale_shape{32, 32, 32};
+
+    const auto query = std::make_shared<ov::op::v0::Parameter>(element::f32, query_key_value_mask_scale_shape);
+    const auto key = std::make_shared<ov::op::v0::Parameter>(element::f32, query_key_value_mask_scale_shape);
+    const auto value = std::make_shared<ov::op::v0::Parameter>(element::f32, query_key_value_mask_scale_shape);
+    const auto attention_mask = std::make_shared<ov::op::v0::Parameter>(element::f32, query_key_value_mask_scale_shape);
+    const auto scale = std::make_shared<ov::op::v0::Parameter>(element::f32, query_key_value_mask_scale_shape);
```

Comment:
+ other scale inputs - according to specification, scale should be either scalar or 1-element 1d tensor

Reply:
Right, changed to match the specs